README.md

```diff
@@ -0,0 +1,22 @@
+# Canonical livepatch machine charm
```

Comment:
**nitpick/style:** Consider captialising 'Livepatch' throughout. It looks better imo.

Reply:
**nitpick/style:** Consider adding the following at the top of the repo:

```yaml
[![CharmHub Badge](https://charmhub.io/livepatch/badge.svg)](https://charmhub.io/livepatch)
[![Release](https://github.com/canonical/livepatch-machine-charm/actions/workflows/release.yaml/badge.svg)](https://github.com/canonical/livepatch-machine-charm/actions/workflows/release.yaml)
[![Tests](https://github.com/canonical/livepatch-machine-charm/actions/workflows/ci.yaml/badge.svg?branch=main)](https://github.com/canonical/livepatch-machine-charm/actions/workflows/ci.yaml?query=branch%3Amain)
[![Docs](https://github.com/canonical/livepatch-machine-charm/actions/workflows/sync_docs.yaml/badge.svg)](https://github.com/canonical/livepatch-machine-charm/actions/workflows/sync_docs.yaml)
```

NOTE - The links probably aren't correct, but hopefully they are self-explanatory enough to be replaced by ones relevant to your charm.

Reply:
**nitpick/style:** Depending on the licensing and ownership of the product, you may be allowed to add logos and branding images. This goes a long way in legitimising the codebase.

Reply:
I fixed the casing you mentioned, and also added the badges that apply. But for the logo and the organized docs, we don't have them yet, and as this is an optional thing, I think we'd like to proceed without them for now.